eng/packaging.targets

```diff
@@ -17,6 +17,16 @@
     <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
     <LibIntellisenseDocumentationFilePath>$(XmlDocFileRoot)1033\$(AssemblyName).xml</LibIntellisenseDocumentationFilePath>
     <UseIntellisenseDocumentationFile Condition="'$(UseIntellisenseDocumentationFile)' == '' and Exists('$(LibIntellisenseDocumentationFilePath)')">true</UseIntellisenseDocumentationFile>
+    <IsAspNetCoreApp Condition="$(AspNetCoreAppLibrary.Contains('$(AssemblyName);'))">true</IsAspNetCoreApp>
```

Comment:
This should be moved down into the property group below.

Reply:
We need it to execute in non-servicing builds as well

Reply:
Oh that's because of binplacing, I see. Fine then. I will get rid of the binplacing in a subsequent PR.